web-app/js/angular/directives/publication.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Created by Temi Varghese on 15/07/15.
+ */
+profileEditor.directive('publication', function ($browser) {
+    return {
+        restrict: 'AE',
+        require: [],
+        scope: {
+            authors: '=',
+            description: '=',
+            doi: '=',
+            publicationDate: '=',
+            title: '=',
+            uploadDate: '=',
+            uuid: '=',
+            profileId: '=',
+            opusId: '=',
+            context: '=',
```

Comment:
The context path is already available in the util class. Inject that as a dependency rather than specifying the context in the directive.